docs/src/stories/Simple/index.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+import DropdownTreeSelect from '../../../../src'
+import '../../../../dist/styles.css'
+
+import './index.css'
+import data from './data.json'
+
+const onChange = (curNode, selectedNodes) => {
```

Comment:
maybe we could factorize those parts of code between stories

Reply:
We could, but the initial thinking was to keep things simple, self contained so that it's easier to grasp for newcomers.